packages/execution-environments/src/web-workers/index.ts

```diff
@@ -0,0 +1,15 @@
+//------------------------
+// Notice that, compared to the rest of the package, we're including ses directly here, instead of just referencing types.
+// This is because IFrame has ses included as a seperate <script> tag, while WebWorkers is just one file.
+// We ensure ses is the first thing to be executed in the index file of WebWorker code
+/* eslint-disable-next-line import/no-unassigned-import */
+import 'ses';
```

Comment:
we shouldnt include this in the bundle see https://github.com/MetaMask/snaps-skunkworks/pull/231/files#r807333562